qiskit_metal/renderers/renderer_ansys/ansys_renderer.py

```diff
@@ -179,41 +213,125 @@ class QAnsysRenderer(QRenderer):
-            render_template (Dict, optional): Typically used by GUI for template options for GDS. Defaults to None.
-            render_options (Dict, optional):  Used to override all options. Defaults to None.
+            options (Dict, optional):  Used to override all options. Defaults to None.
         """
-        super().__init__(design=design,
-                         initiate=initiate,
-                         render_template=render_template,
-                         render_options=render_options)
+        super().__init__(design=design, initiate=initiate, options=options)
 
         # Default behavior is to render all components unless a strict subset was chosen
         self.render_everything = True
 
         self._pinfo = None
+        # Connected to Ansys variables
+        self._rapp = None
+        self._rdesktop = None
+
+    @property
+    def initialized(self):
+        if self._pinfo:
+            if self._pinfo.project:
+                try:
+                    # this is because after previous pyEPR close(),
+                    # the pinfo.project becomes None, but the type is not None (method)
+                    # TODO: fix where appropriate, then remove this patch.
+                    self._pinfo.project.name
+                except AttributeError:
+                    return False
+                return True
+        return False
+
+    @property
+    def rapp(self):
+        return self._rapp
+
+    @rapp.setter
+    def rapp(self, app_com):
+        if self._rapp:
+            self._rapp.release()
+        self._rapp = app_com
+
+    @property
+    def rdesktop(self):
+        return self._rdesktop
+
+    @rdesktop.setter
+    def rdesktop(self, desktop_com):
+        if self._rdesktop:
+            self._rdesktop.release()
+        self._rdesktop = desktop_com
+
+    def _initiate_renderer(self):
+        """
+        Open a session of the default Ansys EDT.
+        Establishes the connection to the App and Desktop only.
+        """
+        # test if ansys is open
+        # import psutil
+        # booted = False
+        # for proc in psutil.process_iter():
+        #     if 'ansysedt' in proc.name():
+        #         booted = True
+
+        # if not booted:
+        #    self._open_ansys(*args, **kwargs)
+        # need to make it so that it waits for the Ansys boot to end
+        # after opening, should establish a connection (able to create a new project)
+
+        self.rapp = HfssApp()
+        self.rdesktop = self.rapp.get_app_desktop()
+        if self.rdesktop.project_count() == 0:
+            self.rdesktop.new_project()
+        self.connect_ansys()  # TODO: can this be done differently?
+
+        # return True to indicate successful completion
+        return True
+
+    def _close_renderer(self):
+        """Not used by the gds renderer at this time. only returns True
+
+        Returns:
+            bool: True
```

Comment:
If this always returns true, why return anything at all?

Reply:
Not sure. I believe I have put that boolean return value as a placeholder. eventually it should return false if something goes wrong, jut did not implement the error management